R/checks.R

```diff
@@ -169,7 +169,7 @@ check_file_details = function(relative_paths){
   not_found = c()
   base_path = check_config_value(config::get("project_base"))
   for(relative_path in relative_paths){
-    full_path = paste0(base_path,relative_path)
+    full_path = file.path(base_path, relative_path)
```

Comment:
Do you happen to have an example of the error this was generating? The `paste0` approach is used extensively to get the path to file and hasn't been failing so far - so wondering to check hot it was behaving here.

Reply:
Got it, thanks for the example. Will it still work as expected if the user will specify the project_base correctly, with trailing `/` ? Just want to confirm that the presence of `//` when this function is used won't cause any issues downstream

Reply:
Yes, it works. When I run `check_gamblr_config` to check if there are missing files (it's not `check_file_details` which does that, it is called internally and was where I made the change), files with the `//` pattern could be found. For example:
```
Looking for: /home/vladimir/repos/gambl_results//all_the_things/cnv_master-1.0/merges/genome--hg38.seg
OK
```
It's not beautiful, but robust. If needed, I could add the line `gsub("//", "/", filePath)` to let the function always outputs a single `/` (I would go with that). 

Some more examples to show that paths with `//` works:
```
file.exists("/home/vladimir/repos/gambl_results//all_the_things/cnv_master-1.0/merges/genome--hg38.seg")
# [1] TRUE
readLines("/home/vladimir/repos/gambl_results//all_the_things/cnv_master-1.0/merges/genome--hg38.seg", n = 5)
# [1] "ID\tchrom\tstart\tend\tLOH_flag\tlog.ratio"              "KM-H2\tchr1\t10001\t16849162\t0\t0"
# [3] "KM-H2\tchr1\t16849163\t101772836\t0\t0.5849625007211563" "KM-H2\tchr1\t101772837\t109541401\t0\t0"
# [5] "KM-H2\tchr1\t109541402\t119838793\t1\t0"
```

Reply:
Thanks for confirming, I think it is good then :+1: